packages/custom_lint/test/src/workspace_test.dart

```diff
@@ -714,22 +738,38 @@ void main() {
         );
       }
 
-      test('throws MissingPubspecError if package does not contain a pubspec',
```

Comment:
Why is this test deleted? It should still be valuable from my understanding

Reply:
The test may need some refactoring to still pass, but I would expect this to be an important scenario

Reply:
Added back the test